apps/web/src/server/applications/search/__tests/applications-search.test.js

```diff
@@ -0,0 +1,79 @@
+import { parseHtml } from '@pins/platform';
+import nock from 'nock';
+import supertest from 'supertest';
+import { createTestApplication } from '../../../../../testing/index.js';
+
+const { app, installMockApi, teardown } = createTestApplication();
+const request = supertest(app);
+
+const successFullResponse = {
+	page: 1,
+	pageSize: 1,
+	pageCount: 1,
+	itemCount: 1,
+	items: [
+		{
+			id: 76,
+			reference: null,
+			modifiedDate: '1662373959',
```

Comment:
would this be a string or int?

Reply:
an int usually. not important for this test though